osx/KA-Lite-Monitor/KA-Lite-Monitor.xcodeproj/project.pbxproj

```diff
@@ -286,7 +286,7 @@
 				CLANG_WARN_OBJC_ROOT_CLASS = YES_ERROR;
 				CLANG_WARN_UNREACHABLE_CODE = YES;
 				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
-				CODE_SIGN_IDENTITY = "";
+				CODE_SIGN_IDENTITY = "Developer ID Application: Foundation for Learning Equality, Inc. (H83B64B6AV)";
```

Comment:
@arceduardvincent We don't want to embed the code sign identity on the Xcode project so that it can be built after the installers repo is checked-out by whoever wants to contribute.